### DISCUSSION TITLE
创建Hive外表时报错

### DISCUSSION THREAD

**Discussion by LoganCjn** [apache/doris#18049] [A - General / Q&A]
创建Hive外表时报错：
faild to check if table is exists int XXX.db,org.apache.thrift. TApplicasionException: Invalid method name: ‘get_table_reg‘’
Hive表存在，名称正确，metastore.usir与Hive配置文件一致。
环境是CDH3.6.3

**Comment by LemonLiTree** [comment]
which version?
I think you need to specify the version： 'hive.version' = '1.1.0' in properties
